pyproject.toml

```diff
@@ -174,6 +174,8 @@ pybamm = [
 
 [tool.setuptools.packages.find]
 include = ["pybamm", "pybamm.*"]
+# TODO: remove once https://github.com/pybamm-team/PyBaMM/issues/3480 is resolved
```

Comment:
Unrelated, but do we have and process for periodically checking TODOs to make sure we are taking care of them over time?

Reply:
It should be a good idea to add a workflow to do that. I don't remember the name exactly, but there is a pre-commit hook too that can check for `# TODO:` and `# FIXME:` comments.

Reply:
Also, this `# TODO:` comment in specific should not be added imo, I remember that when I worked on this part of the configuration it was also trying to install the `tests`, `examples`, and the `benchmarks` folders treating them as packages, because they are not namespace packages.